src/webview/MessageListWeb.js

```diff
@@ -15,15 +15,32 @@ import { base64Utf8Encode } from '../utils/encoding';
 export default class MessageListWeb extends Component<Props> {
   context: Context;
   props: Props;
+  isWebViewReady: boolean;
   webview: ?Object;
-  isReady: boolean;
   unsentMessages: WebviewInputMessage[] = [];
 
   static contextTypes = {
     styles: () => null,
     theme: () => null,
   };
 
+  /*
+  *
+  * Synchronizing setup with webview
+  *
+  */
+  sendTestPing = (): void => {
```

Comment:
Rename to one of these:
 * sendHandshake
 * initiateHandshake
 * sendReadyHandshake
 * initiateReadyHandshake

The choice is yours 😄